crates/node/core/src/args/rpc_server.rs

```diff
@@ -156,6 +156,10 @@ pub struct RpcServerArgs {
     )]
     pub rpc_gas_cap: u64,
 
+    /// The maximum proof lookback for historical proof generation.
+    #[arg(long = "rpc.max-proof-lookback", default_value_t = 0)]
+    pub rpc_max_proof_lookback: u64,
```

Comment:
do we want to go with lookback here?
@Rjected any other suggestions? maybe window?